pkg/operator/hive/hive_controller.go

```diff
@@ -69,7 +69,15 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileHiveConfig{Client: mgr.GetClient(), scheme: mgr.GetScheme(), restConfig: mgr.GetConfig(), mgr: mgr}
+	return &ReconcileHiveConfig{
+		Client:     mgr.GetClient(),
+		scheme:     mgr.GetScheme(),
+		restConfig: mgr.GetConfig(),
+		mgr:        mgr,
+		httpProxy:  os.Getenv("HTTP_PROXY"),
+		httpsProxy: os.Getenv("HTTPS_PROXY"),
+		noProxy:    os.Getenv("NO_PROXY"),
+	}
```

Comment:
i think this missed the reconciler to callsite change

Reply:
My bad! Forgot that area entirely.